src/chrome/content/rules/GMX.xml

```diff
@@ -10,45 +9,37 @@
 
 	Nonfunctional domains:
 		- (www.)gmx.ca		(→ www.gmx.com)
-
```

Comment:
You've added back the missing lines, but now they have spaces at the beginning of them, which still generates unnecessary diff lines. I would recommend starting from a fresh branch, making only the necessary changes, and starting a fresh PR.